linux_pipeline/Jenkinsfile_container_images

```diff
@@ -15,14 +35,162 @@ pipeline {
 
   agent {
     node {
-      label 'meta_slave'
+      label 'master'
     }
   }
+
   stages {
-     stage('Noop') {
-       steps {
-         sh 'echo "NOOP"'
-       }
-     }
-  }
-}
\ No newline at end of file
+      stage('Prepare Ubuntu build environment') {
```

Comment:
use prepare_build_env naming for stage names and the same name in the when conditions
# !!! applicable for all stages